text/0000-arguments.md

```diff
@@ -138,31 +145,32 @@ Ensuring all arguments are tokenized and parsed correctly requires some `bash` c
+To take advantage of processes extension via user provided args, buildpacks will need to migrate shell processes from the current model (arguments embedded in the `command` string) to the new one (arguments provided in the `args` array).
 
-To ease this transition we can preserve the existing behavior in the case where there are no arguments provided by either the buildpack process or the user. These processes will break if a user tries to supply additional arguments, but it is already true that users cannot supply additional arguments to these processes.
+To ease this transition, we can preserve the existing behavior in cases where there are no arguments provided by either the buildpack-contributed process definition or dynamically by the user. These processes will break if a user tries to supply additional arguments. However, it is already true that users cannot supply additional arguments to these processes, and therefore no functionality has been removed.
 
 ### Platforms
-We can use `CNB_PLATFORM_API` (API "modes") to toggle between the current and new launcher process resolution flow. `CNB_PLATFORM_API` will be set in the exported image to the platform API provided at build time, but can be overrode explicitly at runtime.
+We can use `CNB_PLATFORM_API` (API "modes") to toggle between the current and new launcher process resolution flow. `CNB_PLATFORM_API` will be set in the exported image to the platform API provided at build time, but can be explicitly overridden at runtime.
+
+The existing `launcher`  behavior wherein it accepts a single positional argument containing a process-types is currently unspecified and undocumented. Current usage of this feature is hard to measure but anecdotally appears to be uncommon.
 
-process-types positional arguments to the `launcher` is currently unspecified and undocumented, current usage of this feature is hard to measure but anecdotally appears to be uncommon.
 
-For the changes proposed in this RFC, changing the default interpretation of `launcher` arguments from a custom command to additional arguments, will likely have the biggest impact on platforms. Platforms will need to make changes to runtime logic and/or documentation before enabling the platform API containing this change.
+Of the changes proposed in this RFC, changing the default interpretation of `launcher` arguments (from a custom command to additional arguments), will likely have the biggest impact on platforms. Platforms will need to make changes to runtime logic and/or documentation before enabling the platform API containing this change.
 
 # Drawbacks
 [drawbacks]: #drawbacks
 
 * Breaking changes!
-* Some functionality is only accessible by setting environment variables, when previously all functionality was available with launcher arguments.
+* Some functionality is only accessible by setting environment variables, when, previously, all functionality was available via the command line.
```

Comment:
I would break it to 2 sentences:
Some functionality is only accessible by setting environment variables. Previously, all functionality was available via the command line.